### PR TITLE
Change of o365 gen script

### PR DESCRIPTION
**Pull request by stavamichal**

### CODE REVIEW

**Comment by stavamichal** [comment]
This pull request just add some changed for pull request #107

File: slave-new/process-o365/conf/pre_01_setup_access_token [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (licehammer) [inline slave-new/process-o365/conf/pre_01_setup_access_token]: Rename this file to example-pre_01_setup_access_token. We want the system administrator to create pre scripts on their own. Also don't want to rewrite any changes in pre scripts with update of DEB/RPM package.

File: slave-new/process-o365/rpm.dependencies [hunk 2]
```diff
@@ -0,0 +1 @@
+perun-slave-base
```
Comment (licehammer) [inline slave-new/process-o365/rpm.dependencies]: Would be nice to add dependencies on used Perl libraries.

**Comment by stavamichal** [comment]
Closing. Final state is in #189.

**Pull request closed without merging**